calico-cloud/threat/container-threat-detection.mdx

```diff
@@ -90,6 +91,29 @@ If a malicious or suspicious program is run within the cluster, it will be repor
 
 ![malware-alert-example](/img/calico-cloud/malware-alert-example.png)
 
+### Create an exception for suspicious programs
+
+If you have a program running within the cluster and want to negate the generation of any runtime logs for that process, thus negating any Security Events for that process, then you can add an entry into the RuntimeExceptionList of the Runtime Security resource through kubectl :
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: operator.tigera.io/v1
+kind: RuntimeSecurity
+metadata:
+  name: default
+spec:
+  runtimeExceptionList:
+    - matching: exact
+      processInvocation: "runc init"
```

Comment:
```suggestion
      processInvocation: "<process-name>"
```
??? If you're using an example, try to have it use a placeholder format: `<placeholder-text>`

Reply:
Otherwise, you can label the entire codeblock:
````bash title="Example RuntimeSecurity CR with exceptions" `
And add explanations about the specific names.